revscoring/features/diff.py

```diff
@@ -133,10 +131,13 @@ def process_uppercase_chars_removed(diff_removed_segments):
         modifiers.max(parent_revision.proportion_of_uppercase_chars, 1)
 
 def process_longest_repeated_char_added(diff_added_segments):
-    return      max(sum(1 for _ in group)
-                    for segment in diff_added_segments
-                    for _, group in groupby(segment.lower()))
-
+    try:
+        return max(sum(1 for _ in group)
+                   for segment in diff_added_segments
+                   for _, group in groupby(segment.lower()))
+    except ValueError:
+        # Happens when there's no segments added
+        return 0
```

Comment:
Isn't 1 a better value than 0 in this case? E.g. If I add "foo", the longest repeated char added was "o", which appeared 2 times. However, if I add "bar", each of the characters was added exactly 1 time.